### ISSUE TITLE
Image region as polygon: error on change size

### ISSUE THREAD

**Issue by regexgit** [jim-easterbrook/Photini#248]
Debian 12, pyside6-6.7.2, Photini 2024.5.0

Image region / new polygon: move is OK but on dragging small rectangle corner I get error:

```python
00:10:26: ERROR: photini.pyqt: 'PySide6.QtGui.QPolygonF' object has no attribute 'replace'
Traceback (most recent call last):
  File "/data/applis/venv_python/photini/lib/python3.11/site-packages/photini/pyqt.py", line 190, in wrapper
    return func(*args, **kwds)
           ^^^^^^^^^^^^^^^^^^^
  File "/data/applis/venv_python/photini/lib/python3.11/site-packages/photini/regions.py", line 54, in mouseMoveEvent
    self.parentItem().handle_drag(self, self.pos())
  File "/data/applis/venv_python/photini/lib/python3.11/site-packages/photini/regions.py", line 350, in handle_drag
    polygon.replace(idx, pos)
    ^^^^^^^^^^^^^^^
AttributeError: 'PySide6.QtGui.QPolygonF' object has no attribute 'replace'
```

**Comment by jim-easterbrook** [comment]
I suspect this is another PySide / PyQt difference. Can you post the output of `python -m photini --version`.

**Comment by jim-easterbrook** [comment]
OK, I can reproduce the bug here, but only with PySide6 (v6.4.0). PyQt5/6 and PySide2 are unaffected.

**Comment by jim-easterbrook** [comment]
`replace` is definitely a member of Qt's `QPolygonF` (inherited from `QList`):
https://doc-snapshots.qt.io/qt6-6.8/qpolygonf-members.html

`replace` appears to be missing from PySide6's version of `QPolygonF`:
https://doc.qt.io/qtforpython-6/PySide6/QtGui/QPolygonF.html

The solution, which seems to work for all Python Qt packages, is to replace `polygon.replace(idx, pos)` with `polygon[idx] = pos`. This will be in the next release, in a week or two.

**Comment by jim-easterbrook** [comment]
I've just released version 2024.8.0 that should fix this and other problems.
